gen-kdump-sysconfig.sh

```diff
@@ -90,13 +94,17 @@ ppc64)
 		"hugepages hugepagesz slub_debug quiet log_buf_len swiotlb hugetlb_cma ignition.firstboot"
 	update_param KDUMP_COMMANDLINE_APPEND \
 		"irqpoll maxcpus=1 noirqdistrib reset_devices cgroup_disable=memory numa=off udev.children-max=2 ehea.use_mcs=0 panic=10 kvm_cma_resv_ratio=0 transparent_hugepage=never novmcoredd hugetlb_cma=0"
+	update_param FADUMP_COMMANDLINE_APPEND \
+		"nr_cpus=16 numa=off cgroup_disable=memory cma=0 kvm_cma_resv_ratio=0 hugetlb_cma=0 transparent_hugepage=never novmcoredd udev.children-max=2"
```

Comment:
Is ppc big-endian supported yet? Since maxcpus parameter is not change for that arch. So I prefer to keep ppc case unchanged.

Otherwise, LGTM.


Thanks,

Pingfan

Reply:
Hello Pingfan, thanks for the review.
The pass additional parameters support applies to both big endian and little endian variants.
Also, nr_cpus support has always been there for fadump but there was no practical way to pass additional parameters to dump capture kernel for fadump till kernel commit 3416c9daa6b13 ("powerpc/fadump: pass additional parameters when fadump is active") made it possible..

Reply:
Hello Hari, my point is there is no support for ppc64be on either Fedora or RHEL. So the branch of ppc64be will be cleaned up later, and no new changes to it any more.

Thanks

Reply:
Ah! You mean, no changes to ppc64. Fine by me. Updated the pull request dropping the ppc64 hunk..